src/webgpu/shader/execution/builtin/builtin.ts

```diff
@@ -11,7 +11,7 @@ import { ScalarType } from '../../types.js';
 
 export const g = makeTestGroup(GPUTest);
 
-export enum OperandType {
+export enum NumberType {
```

Comment:
I'm thinking of changing this enum to a bool values: isHex.
Currently if the NumberType is anything other that the baseType it has to be Hex.
I'm not sure if later we are going to use Uint numberTypes to test a builtin with i32 operands (baseType will be i32).
@ben-clayton Please let me know what you think.

Reply:
It occurs to me I didn't really understand what this enum means, or what `isHex` would mean. I think I understood when I reviewed before, but I had to go looking at code usages to figure it out again. IIUC it essentially defines how the input/expected values are interpreted: as a number or as a bit representation. So the boolean could be something like `reinterpretBits` or `isBitRepresentation`.

I also wonder if we can eventually encapsulate this better. I started taking a stab at it for just this use-case: #747 (though perhaps someday should be encapsulated more generally along with the texel representation stuff in `webgpu/util/texel/texel_data.ts` which more generally deals with representing numbers in different ways).

If you think it would help, feel free to review/land/use #747.

Reply:
I think #747 is the cleanest option here. +1 from me.